### PR TITLE
prop-types cleanup

### PR DESCRIPTION
**Pull request by zburke**
Prop-types clean up during some React v18 testing. Could. Not. Resist.

### CODE REVIEW

**Comment by Terala-Priyanka** [comment]
Hey @zburke 
hope you would fix failures.

**Comment by zburke** [comment]
Whoops, sorry, @Terala-Priyanka. I didn't realize how react-intl was mocked in that one test. Fixed!

File: src/settings/CommentRequiredForm.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const Setting = ({
 );
 
 Setting.propTypes = {
-  label: PropTypes.element,
+  label: PropTypes.string,
```
Comment (Terala-Priyanka) [inline src/settings/CommentRequiredForm.js:48]: Hey Zak,
I see the reason underlying in here (Select accepts aria-label prop of type string), that made you update label prop type as string.
Wouldn't eslint complaint in this case? How did it go unaddressed till now?

**Review by Terala-Priyanka — APPROVED** [review]

File: src/settings/CommentRequiredForm.js (hunk 1 above)
Comment (zburke) [reply src/settings/CommentRequiredForm.js:48]: ESLint is only going to do static analysis so it won't pick up this kind of thing.

**Pull request merged**